testing/test_pdb.py

```diff
@@ -3726,6 +3747,10 @@ def check_completions():
 """)
 
 
+@pytest.mark.skipif(
+    os.getenv("APPVEYOR", False),
+    reason="Fails in appveyor but not local Windows. Can't figure out why."
+)
```

Comment:
Both variants, or only the "pyrepl" one?
Can you provide more info, e.g. the full output?

Reply:
Hmm... well it was both variants, but I think that may have just been an issue with my implementation. It appears to be working again, so I'll remove the skipif.